extensions/wikia/EditPageLayout/js/plugins/TemplateClassificationEditorPlugin.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Editor plugin to force template classification type choose on publish if doesn't exist and break article submit
+ *
+ * @author Kamil Koterba <kamil@wikia-inc.com>
+ */
+(function (window, $) {
+		'use strict';
+
+		var WE = window.WikiaEditor = window.WikiaEditor || (new window.Observable()),
```

Comment:
is window.observable commint from our own modules / external libs? (just out of curiosity)